SwiftUIDZ/SwiftUIDZ/Filters/FiltersView.swift

```diff
@@ -0,0 +1,213 @@
+//
+//  FiltersView.swift
+//  SwiftUIDZ
+//
+//  Created by Алена Панченко on 05.02.2023.
+//
+
+import SwiftUI
+
+/// Экран  фильтра
+struct FiltersView: View {
+    
+    // MARK: - Private Constants
+    
+    private enum Constants {
+        static let emptyString = ""
+        static let alertText = "Settings apply"
+        static let updateButtonTitleText = "Update Settings"
+        static let chevronLeftSystemImageName = "chevron.left"
+        static let chevronRightSystemImageName = "chevron.right"
+        static let toolBarItemText = "Filters"
+        static let gradientColorTwoName = "gr"
+        static let filterImageName = "filt"
+        static let categoryText = "Category"
+        static let MoreText = "More"
+        static let colorText = "Color"
+        static let pricingText = "Pricing"
+    }
+    
+    // MARK: - Public Properties
+    
+    var body: some View {
+        VStack {
+            gradientNavigationView
+            VStack {
+                categoryView
+                furnituresLazyHGridView
+                    .offset(y: -70)
+                pricingSliderValue
+                colorTextView
+                colorsLazyVGridView
+                Spacer()
+            }
+        }
+        .hiddenTabBar()
+        .toolbar {
+            ToolbarItem(placement: .principal) {
+                toolbarNotificationTitleTextView
+            }
+        }
+        .navigationBarBackButtonHidden()
+        .navigationBarItems(leading: backButtonView)
+        .navigationBarItems(trailing: filterButtonView)
+    }
+    
+    // MARK: - Private Properties
+    
+    private let rows: [GridItem] = [
+        .init(.fixed(250))]
+    
+    private let columns: [GridItem] = [
+        .init(.flexible()),
+        .init(.flexible()),
+        .init(.flexible()),
+        .init(.flexible()),
+        .init(.flexible())
+    ]
+    
+    @Environment(\.presentationMode) var presentation
+    
+    @StateObject private var filtersViewModel = FilterViewModel()
+    
+    private var pricingSliderValue: some View {
+        VStack {
+            HStack {
+                Text(Constants.pricingText)
+                    .font(.largeTitle)
+                    .foregroundColor(.black.opacity(0.7))
+                    .bold()
+                Spacer()
+            }
+            .padding(.leading, 20)
+            RangedSliderView(value: $filtersViewModel.sliderPosition, bounds: 100...5000)
+                .padding(.horizontal, 50)
+        }
+        .offset(y: -70)
+    }
+    
+    private var colorTextView: some View {
+        HStack {
+            Text(Constants.colorText)
+                .foregroundColor(filtersViewModel.currentColor)
+                .font(.largeTitle)
+                .bold()
+            Spacer()
+        }
+        .padding(.horizontal, 20)
+    }
+    
+    private var colorsLazyVGridView: some View {
```

Comment:
```suggestion
    private var colorsLazyVGridView: some View {
```
Был разговор, что так больше не называем... теперича типа colorsView